### PR TITLE
Backend refix routing

### PR DESCRIPTION
**Pull request by bright258**
Fixed routes, migrations, and authentication for good

### CODE REVIEW

File: project-backend/src/controllers/auth/auth.controller.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Controller, Post, Request } from '@nestjs/common';
+import { AuthService } from '../../services/auth/auth.service';
+
+@Controller('auth')
+export class AuthController {
+  constructor(private authService: AuthService) {}
+
+  @Post('/auth')
```
Comment (3cognito) [inline project-backend/src/controllers/auth/auth.controller.ts]: auth/auth?

Why not @Post('/login'). Clearer what is happening

File: project-backend/src/modules/auth/auth.constants.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export const jwtSecret =
+  'Well this is something really difficult to remember 23440@!FPAaspoj';
```
Comment (3cognito) [inline project-backend/src/modules/auth/auth.constants.ts]: should be in your environment variable file and not committed to VCS

File: project-backend/src/services/auth/auth.service.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import * as bcrypt from 'bcrypt';
+
+@Injectable()
+export class AuthService {
+  constructor(
+    private userService: UserService,
+    private jwtService: JwtService,
+  ) {}
+
+  async signIn(username: string, password: string) {
```
Comment (3cognito) [inline project-backend/src/services/auth/auth.service.ts]: No error thrown for incorrect credentials?

**Review by 3cognito — APPROVED** [review]

**Comment by bright258** [comment]
oh, I thought  throwing unauthorized exception would suffice. Thanks for pointing that out, I'll factor it in

**Pull request merged**